eitprocessing/mixins/slicing.py

```diff
@@ -53,50 +42,55 @@ def __getitem__(self, key: slice | int):
             )
 
         if isinstance(key, int):
-            return self.select_by_index(start=key, end=key, end_inclusive=True)
+            return self.select_by_index(
+                start=key, end=key, start_inclusive=True, end_inclusive=True
+            )
 
-        raise TypeError(f"Invalid key type. Should be slice or int, not {type(key)}.")
+        raise TypeError(
+            f"Invalid slicing input. Should be `slice` or `int`, not {type(key)}."
+        )
 
     @abstractmethod
     def _sliced_copy(
```

Comment:
This method does not accomplish anything. It does not return any value or
change anything about self. Maybe you could create a different function to
create a label and use that?

```python
    def _create_label_name(start_index, end_index):
        ...
```